docs/status-and-issues.md

```diff
@@ -28,43 +28,47 @@ All of these configurations use Pulumi code within Python as the Infrastructure
+   currently in active development and will result in this process being streamlined for these additional environments.
+   Please follow
+   [#80](https://github.com/nginxinc/kic-reference-architectures/issues/80) for details.
+4. We are currently using filebeat as our logging agent. This deployment requires that the correct paths to the
+   container log directory are present in the deployment data. We have discovered that this differs based on the K8
+   provider. Please see [#76](https://github.com/nginxinc/kic-reference-architectures/issues/76) for more detail.
 
 ## Known Issues / Caveats
+
 1. Currently, the use of the Elastic tooling has shown to be problematic under heavy load, with containers falling over
-and causing disruptions. Please see the tuning variables in the configuration file to adjust the number of replicas 
-deployed for the Elastic logstore to tune to your environment.
-2. The default Helm timeout is 5 minutes, which is acceptable for most managed clouds but tends to be too short for 
-single-vm or workstation deployments. Please see the configuration file variables to adjust the helm timeout as required
-for your environment.
+   and causing disruptions. Please see the tuning variables in the configuration file to adjust the number of replicas
+   deployed for the Elastic logstore to tune to your environment.
+2. The default Helm timeout is 5 minutes, which is acceptable for most managed clouds but tends to be too short for
+   single-vm or workstation deployments. Please see the configuration file variables to adjust the helm timeout as
+   required for your environment.
 3. When load testing the Bank of Sirius using Locust, you will likely see a high failure rate as you increase the max
-users and spawn rate. This is "normal" and is an area we want to expose and explore for troubleshooting, determining
-which metrics/traces are helpful, etc.
+   users and spawn rate. This is "normal" and is an area we want to expose and explore for troubleshooting, determining
+   which metrics/traces are helpful, etc.
 4. The most common failure modes for non-cloud environments tend towards the following failures:
-   1. Unable to provision persistent storage; correct by ensuring you have a persistent volume provider and can provision a volume.
-   2. Unable to provision an External IP; correct by adding an IP provider such as kubevip or metallb.
-   3. Resource starvation (not enough CPU, Memory); expand the size of the VM or detune the environment.
-   4. Timeouts in helm; increase the helm timeout in the configuration file.
-5. If you are using a cloud provider with timed credentials, such as AWS, one failure mode that can arise is when the 
-credentials expire. This will result in a number of strange and seemingly confusing errors. Double check to make sure that
-the credentials are valid.
+    1. Unable to provision persistent storage; correct by ensuring you have a persistent volume provider and can
```

Comment:
Is there a reasonable tool you could suggest here as an example.  I'm thinking about keywords to kick people off googling the right thing if `volume provider` does not immediately make sense to them.

Reply:
Very good point; I included a number of links - on PV's, metallb, kubevip, locust - that should help the discerning user dig further.